src/components/tab/tab.jsx

```diff
@@ -56,6 +63,8 @@ Tab.propTypes = {
   singlePanel: PropTypes.bool,
   changeHandler: PropTypes.func,
   className: PropTypes.string,
+  node: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+  to: PropTypes.string,
```

Comment:
Gonna want a default on this one

Reply:
From our eslint rules:

>> Always define explicit defaultProps for all non-required props.

>Why? propTypes are a form of documentation, and providing defaultProps means the reader of your code doesn’t have to assume as much. In addition, it can mean that your code can omit certain type checks.

https://github.com/airbnb/javascript/tree/master/react#props

Reply:
I think this rule came after the version we have in the ui kit as of now, so the linting didn't catch it, but the reason for having it still stands.